xbmc/music/ContextMenus.cpp

```diff
@@ -21,11 +24,59 @@ CMusicInfo::CMusicInfo(MediaType mediaType)
   : CStaticContextMenuAction(19033), m_mediaType(std::move(mediaType))
 {
 }
+namespace
```

Comment:
intended to nest the anonymous namespace inside CONTEXTMENU? looks a bit odd to me, can be ok though

Reply:
Video context menu does a similar thing I believe.